src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnAndAatfToSentOnRemoveSiteViewModelMap.cs

```diff
@@ -31,7 +31,9 @@ public SentOnRemoveSiteViewModel Map(ReturnAndAatfToSentOnRemoveSiteViewModelMap
                 ReturnId = source.ReturnId,
                 AatfId = source.AatfId,
                 SiteAddress = addressUtilities.FormattedAddress(source.WeeeSentOn.SiteAddress),
```

Comment:
@sreedhar-civica Could we update ReturnAndAatfToSentOnRemoveSiteViewModelMapTests to check for the setting of the new properties...